proto/il/v1/tx.proto

```diff
@@ -0,0 +1,28 @@
+syntax = "proto3";
+package il.v1;
+
+import "il/v1/il.proto";
+
+option go_package = "github.com/peggyjv/sommelier/x/il/types";
+
+// MsgService defines the msgs that the oracle module handles.
+service Msg {
+  rpc CreateStoploss  (MsgStoploss)   returns (MsgStoplossResponse);
+  rpc DeleteStoploss  (MsgDeleteStoploss)   returns (MsgDeleteStoplossResponse);
+}
+
+// MsgStoploss defines a stoploss position
+message MsgStoploss {
+    string address = 1;
+    Stoploss stoploss = 2;
```

Comment:
One thought, maybe we want the user to not submit the reference pair ratio, and/or we check it/set it using the oracle data